setup.py

```diff
@@ -48,7 +48,7 @@
 ]
 
 setup(name="mediagrains",
-      version="2.8.3",
+      version="2.8.4",
```

Comment:
Probably should be "2.9.0.dev1", I think.

Reply:
Yup, you make a very valid point about the dev branch. 0f6ca58 fixes the version number and I've re-targeted the PR